app/v2/otherCgt/createAmend/CreateAmendOtherCgtConnector.scala

```diff
@@ -37,13 +37,19 @@ class CreateAmendOtherCgtConnector @Inject() (val http: HttpClientV2, val appCon
     import request.*
     import shared.connectors.httpparsers.StandardDownstreamHttpParser.*
 
-    val downstreamUri: DownstreamUri[Unit] =
-      if (taxYear.useTaxYearSpecificApi) {
-        IfsUri[Unit](s"income-tax/income/disposals/other-gains/${taxYear.asTysDownstream}/${nino.nino}")
+    lazy val downstreamUri1886: DownstreamUri[Unit] =
+      if (ConfigFeatureSwitches().isEnabled("ifs_hip_migration_1886")) {
+        HipUri[Unit](s"itsa/income-tax/v2/${taxYear.asTysDownstream}/income/disposals/other-gains/${nino.nino}")
```

Comment:
I'm currently reviewing the stub but seeing test failures that are caused by this - I think this should still be "v1" in the downstream URI as it needs to refer to the version of the downstream API (which is v1) rather than the version of our API
```suggestion
        HipUri[Unit](s"itsa/income-tax/v1/${taxYear.asTysDownstream}/income/disposals/other-gains/${nino.nino}")
```